### PR TITLE
Fix Accordion interaction test example

### PR DESCRIPTION
**Pull request by Tiendq**
Fixed several issues in old test example (it's actually copied from current unit test code with a little modification):

- Include entire `it` function so it's easier to understand, especially know where `done` is from.
- Remove redundant ');'
- to.be.true is more suitable with `isExpanded`
- `wrapper` instead of `tree`

### CODE REVIEW

**Comment by lancewillett** [comment]
Thanks for contributing! Left a minor code comment to match our JavaScript style guide: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md

**Comment by Tiendq** [comment]
Hi @lancewillett,

I just added some comments. Thanks.

**Comment by Tiendq** [comment]
@gwwar Can you have a look at it? Thanks.

File: docs/react-component-unit-testing.md [hunk 1]
```diff
@@ -54,19 +54,23 @@ Example test from `client/components/Accordion`
 ```javascript
 import { shallow } from 'enzyme';
```
Comment (gwwar) [inline docs/react-component-unit-testing.md]: Thanks @Tiendq changes here look good.

Maybe also add:
```javascript
import { expect } from 'chai';
```

Comment (Tiendq) [inline docs/react-component-unit-testing.md]: @gwwar Added, thank you.

File: docs/react-component-unit-testing.md [hunk 2]
```diff
@@ -54,19 +54,20 @@ Example test from `client/components/Accordion`
...
+it( 'should toggled', function( done ) {
+  const wrapper = shallow( <Accordion title="Section" onToggle={ finishTest }>Content</Accordion> );
 
-wrapper.find( '.accordion__toggle' ).simulate( 'click' );
-);
+  wrapper.find( '.accordion__toggle' ).simulate( 'click' );
 
-function finishTest( isExpanded ) {
-	expect( isExpanded ).to.be.ok;
+  function finishTest( isExpanded ) {
```
Comment (Tiendq) [inline docs/react-component-unit-testing.md]: @lancewillett I fixed tabs issue. Thank you.

**Pull request merged**